lib/pages/notifications/set_notification_widget.dart

```diff
@@ -0,0 +1,81 @@
+// ignore_for_file: prefer_const_constructors
+
+import 'package:flutter/material.dart';
+import 'package:mazilon/pages/notifications/notification_service.dart';
+import 'package:mazilon/pages/notifications/time_picker.dart';
+import 'package:mazilon/util/appInformation.dart';
+import 'package:mazilon/util/userInformation.dart';
+import 'package:provider/provider.dart';
+
+class SetNotificationWidget extends StatefulWidget {
+  const SetNotificationWidget({super.key});
+
+  @override
+  State<SetNotificationWidget> createState() => _SetNotificationWidgetState();
+}
+
+class _SetNotificationWidgetState extends State<SetNotificationWidget> {
+  int _currentHour = 12;
+  int _currentMinute = 0;
+  void setTime(int minute, int hour) {
```

Comment:
Please set the initial time to the time the user set (unless it is the first time)